ui/packages/catalog-ui-search/src/main/webapp/js/cql.js

```diff
@@ -581,6 +581,9 @@ function write(filter) {
       }
       break
     case comparisonClass:
+      if (filter.type === 'IS NULL') {
+        return '("' + filter.property + '" ' + filter.type + ')'
```

Comment:
This could be a template literal (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)